ipaserver/install/dsinstance.py

```diff
@@ -390,7 +390,9 @@ def create_replica(self, realm_name, master_fqdn, fqdn,
 
         self.step("creating DS keytab", self._request_service_keytab)
         if self.promote:
-            if self.ca_is_configured:
+            if self.pkcs12_info:
+                self.step("configuring ssl for ds instance", self.__enable_ssl)
```

Comment:
There are other occurrences of lower-case "ssl" and "ds" within strings in this file.  (The string above is a copy-paste of another).  I created a new ticket: https://fedorahosted.org/freeipa/ticket/6586.  I will add a commit to this PR to address it.